src/jfem.jl

```diff
@@ -103,10 +103,15 @@ atan(u::Union{Expression,FeFunction}) = Expression(fenics.atan(u.pyobject))
 exp(u::Union{Expression,FeFunction}) = Expression(fenics.exp(u.pyobject))
 log(u::Union{Expression,FeFunction}) = Expression(fenics.ln(u.pyobject))
 
+bessel_I(nu::Int, u::Union{Expression,FeFunction}) = Expression(fenics.bessel_I(nu, u.pyobject))
```

Comment:
These need to import from SpecialFunctions.jl?

Reply:
I had the same thought, but wasn't too sure. SpecialFunctions.jl is a third-party library that is not (yet) part of Julia's standard library. So if we import those functions from there we would be adding another dependency to FEniCS.jl. If that is what you think is best I will do just that and rename the functions I just added in order to remains consistent with the naming in SpecialFunctions.jl

Reply:
It's a standard library. It's incorrect to think of things like SpecialFunctions.jl as somehow different from the standard library because it's versioned outside the language itself: it's pretty much in every installation already.

Reply:
Okay, I was not aware of that. I actually tried to look up SpecialFunctions.jl in the Julia standard library section of the documentation. Because I did not encounter it I just assumed. I will apply the modifications in a moment.

Reply:
Done